### PR TITLE
Clean cli output

### LINKED ISSUE #40: Do not log anything by default
The default should not log data in stdout. It should be only enabled by a flag and a CLI option. Any error should only be in stderr anyway otherwise a `--json -` will report damaged JSON.

### PR DESCRIPTION
**Pull request by TG1999**
closes #40

### CODE REVIEW

File: src/python_inspector/resolution.py [hunk 1]
```diff
@@ -516,17 +523,22 @@ def get_resolved_dependencies(
...
+                environment=environment, repos=repos, resolved_requirements=resolved_requirements
+            ),
+            reporter=BaseReporter(),
+        )
+        results = resolver.resolve(requirements=requirements, max_rounds=max_rounds)
+        results = format_resolution(results, as_tree=as_tree, environment=environment, repos=repos)
+        return results
+    except Exception as e:
+        if debug:
+            click.secho(f"{e!r}", err=True)
```
Comment (pombredanne) [inline src/python_inspector/resolution.py:543]:
```suggestion
            import click
            click.secho(f"{e!r}", err=True)
```

File: src/python_inspector/resolution.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 from typing import Sequence
 from zipfile import ZipFile
 
+import click
```
Comment (pombredanne) [inline src/python_inspector/resolution.py:18]: Let's hide this import in the debug block
```suggestion
```

**Review by pombredanne — APPROVED** [review]
LGTM with a minor nit... but there seems to be changes already covered in #42 ? Are these the same?

**Comment by TG1999** [comment]
I have cut a branch from https://github.com/nexB/python-inspector/pull/42 here, will rebase once https://github.com/nexB/python-inspector/pull/42 gets merged

**Comment by TG1999** [comment]
Thanks! merging

**Pull request merged**